src/Component/Native/Workbench/index.js

```diff
@@ -301,6 +312,19 @@ export default class ComponentWorkbench {
     return this.on(CHANGE_TOPIC, callback);
   }
 
+  // visibility changes are issued _before_ renderer.setContainer() is called to render the viewport's contents
+  // if index is -1, viewport will not be in the DOM
+  // if index is >= count, viewport is in the DOM but not visible in the current layout
+  triggerVisibilityChange(renderer, index, layout) {
```

Comment:
renderer is an internal name.

The api use viewport instance, not renderer. (https://github.com/Kitware/paraviewweb/blob/f2b05b822a6898813827d899551f2db62154d97c/src/Component/Native/Workbench/index.js#L197)